lib/new_inject.rb

```diff
@@ -0,0 +1,7 @@
+class Array
+
+  def new_inject(&block)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used. You can also write as `new_inject(*)` if you want the method to accept any arguments but don't care about them.